src/libmongoc/src/mongoc/mongoc-cluster-aws.c

```diff
@@ -125,6 +126,7 @@ _sasl_reply_parse_payload_as_bson (const bson_t *reply,
 }
 
 
+#define MONGOC_AWS_SOCKET_TIMEOUT 10000
```

Comment:
Is this used somewhere? Was this meant to be used for `const int socket_timeout_ms = 10000;`?